audio/espeak-ng/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                espeak-ng
+github.setup        espeak-ng espeak-ng 1.50
+version             1.50
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         eSpeak NG is a multi-lingual software speech synthesizer
+
+long_description    eSpeak NG is a multi-lingual software speech synthesizer
+
+homepage            https://github.com/espeak-ng/espeak-ng
+distname            ${version}
+checksums           rmd160  bd3dd4e17863bb1aa3a964517d888ed9997f0e3e \
+                    sha256  f0505366d5391f1291a9c5cc0816461719f46eaed84cfbf57fb81b0dc381de21 \
+                    size    13665736
+
+depends_lib         port:pcaudiolib \
+                    port:sonic \
+                    port:rb-ronn-ng \
+                    port:rb-kramdown \
+                    port:automake \
+                    port:autoconf \
+                    port:libtool \
+                    port:pkgconfig
+
+configure.cmd       ./autogen.sh && ./configure --with-extdict-ru --with-extdict-zh --with-extdict-zhy
+build.cmd           make src/espeak-ng src/speak-ng en && make
+build.target
+build.args-append   -i -k prefix=${prefix}
+destroot.args       {*}${build.args}
+
+test.run            yes
+test.cmd            ESPEAK_DATA_PATH=`pwd` LD_LIBRARY_PATH=src src/espeak-ng "Testing." -w test.wav
```

Comment:
Why set LD_LIBRARY_PATH? macOS doesn't use that variable; it uses DYLD_LIBRARY_PATH.

Normally specifying environment variables to the test phase should be done with test.env. However I had [trouble getting that to work for espeak](https://trac.macports.org/ticket/60722). If the same is happening here, then setting them in test.cmd is fine.